meta-resin-common/recipes-core/systemd/systemd/coredump.conf

```diff
@@ -0,0 +1,2 @@
+[Coredump]
+Storage=none
```

Comment:
We can limit the coredumps as well. We just have to make sure there is enough space for journal, coredumps and other files there. CC @michal-mazurek